packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -928,6 +909,27 @@ export default function makeKernelKeeper(kernelStorage, kernelSlog) {
     kvStore.set(`${kpid}.decider`, '');
   }
 
+  function* enumeratePromisesByDecider(vatID) {
```

Comment:
@warner We talked about a convention for generators, and "enumerate…" has some preexisting examples so I'm running with it.